.travis.yml

```diff
@@ -2,57 +2,36 @@ language: node_js
+    # GITHUB_TOKEN
+    - secure: 'VwOddbUrlAn2FLe+DbS28iH0sM7JSXuBmTHxswTNYlM8FDPCVZCV6Hfi9kVuQRmW4EnfWO5HJXOO9fEsRvLJ+TuR0Zia15S0cCNF4AL7S5nGT0gkl/VU1JTCnPKAyHRGp2fNDXjvmjwrqpAPgJArsRoNzT12VASTstQQV47LdwZkJ/5I78Ay1vaULo6urorartqIExzAiEYAlojzrcCZ+sh35Z/dZT3JRr0qA+lBEWc8tfxHaDLwa3YRdVi5VRpRjAy9pSDTIVZGf69zONGR7jMGfMYyF/BI3HCgrjdzatcbp8oACut+eMb6KfGqE1VPWVlBFhp3o3VDOCqsSx46Zins6cHokmrVzcgG9Xfq6Pm4SeRLPSv+3rBEtE1GbWVQoO8PC2X86WweUiSuaknPOLm+lZHi3Y+Stq1E88JLh3hNxMOXhm79dn1/S6dAIcMMkoyFBnmdy2jXoRs0jJ7XruwVSPPCAngOwIfpnZrLuFeAX8AKOrXte65pai/Q+VzZiRHO08EO+GGzfS1nECGEWnVsRXdbRicM5WJLijw1n7/KRLM5lpxgS0kOwzZOEgFgAyiJBz7moNqm/JS2s0B1iMKRh6LLsbVbjQyB3NZLbN1G3X/KUquObn9ZxU55kwqmBvXm7wE4yNqAzGEwsC95S2JsDgdilscNa7D2UzYkPDY='
 cache:
   yarn: true
   directories:
-  - node_modules
-before_install:
-- curl -fsSL https://bootstrap.pypa.io/get-pip.py | python - --user
-- travis_retry pip install --user transifex-client==0.12.5
-- install -m0644 .transifexrc.tpl ~/.transifexrc
-- echo "password = $TX_PASSWD" >> ~/.transifexrc
-stages:
-  - prebuild
-  - build
-jobs:
-  include:
-    - name: 'Lint'
-      stage: 'prebuild'
-      script: yarn lint
-    - name: 'Tests'
-      stage: 'prebuild'
-      script: yarn test
-    - name: 'Build app'
-      stage: 'build'
-      before_install:
-        - curl -fsSL https://bootstrap.pypa.io/get-pip.py | python - --user
-        - travis_retry pip install --user transifex-client==0.12.5
-        - install -m0644 .transifexrc.tpl ~/.transifexrc
-        - echo "password = $TX_PASSWD" >> ~/.transifexrc
-      script: yarn build
-      deploy:
-        - provider: script
-          repo: edas/myapp
-          skip-cleanup: true
-          # deploy the build on a build branch and publish to the Cozy registry
-          script: DEPLOY_BRANCH=build && yarn deploy && yarn cozyPublish
-          on:
-            branch: master
-        - provider: script
-          repo: edas/myapp
-          skip-cleanup: true
-          # publish stable or beta versions using Github Releases (git tag)
-          script: DEPLOY_BRANCH=build && yarn cozyPublish
-          on:
-            tags: true
+    - node_modules
+script:
+  - yarn build
+before_deploy:
+  - yarn add cozy-app-publish
+deploy:
+  - provider: script
+    repo: cozy/cozy-notes
+    skip-cleanup: true
+    # deploy the build on a build branch and publish to the Cozy registry
+    script: DEPLOY_BRANCH=build && yarn deploy && yarn cozyPublish
```

Comment:
it means that if we already have a build in the build branch, this build will be erase.

Should we use something else? Like DEPLOY_BRANCH='auto-build' or `release`?

Reply:
Yep, no problem to change it 👍